src/Components/CategoryReviews.jsx

```diff
@@ -1,19 +1,18 @@
 import React from "react";
 import { useEffect, useState } from "react";
 import { Link, useParams } from "react-router-dom";
-import { FetchAllCategoryReviews } from "./Api";
+import { fetchAllCategoryReviews } from "./Api";
 import "../StyleSheets/GetCategoryReviews.css";
+import ReviewCard from "./ReviewCard";
 
 const GetCategoryReviews = () => {
   const [reviews, setReviews] = useState([]);
   const category = useParams();
 
   useEffect(() => {
-    FetchAllCategoryReviews(category.slug).then((data) =>
+    fetchAllCategoryReviews(category.slug).then((data) =>
       setReviews(data.reviews)
     );
-
-    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
```

Comment:
missing dependency here on line 16 for our use effect where we are fetching reviews by category. When do we want to make this request? what does the error message tell us makes sense to pass in here?